recipes/biodigest/run_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python -m biodigest.single_validation -t target_gene_cluster.txt -ti symbol -m clustering
```

Comment:
Since this is so short, it’s easier to maintain if you don’t add this extra file but keep the test in `meta.yaml`:
```yaml
test:
  commands:
    - python -m ...
```